index.php

```diff
@@ -47,6 +47,7 @@
 $PAGE->set_title($site->fullname);
 $PAGE->set_heading($site->fullname);
 $helper->additional_meta_tags();
+$PAGE->requires->css(new moodle_url('/local/signin/css/style.css'));
```

Comment:
@snehasischatterjee can you ask ben about the inclusion of CSS? I think it may go in the theme.